test/adapters/httpclient_test.rb

```diff
@@ -17,5 +17,7 @@ def test_binds_local_socket
         assert_equal host, conn.options[:bind][:host]
       end
     end
+
+    Integration.apply(self, :Compression)
```

Comment:
You could have added `:Compression` to the existing `Integration.apply` call above in the same file.